cmd/kubelet/app/server.go

```diff
@@ -337,6 +340,16 @@ func (s *KubeletServer) createAPIServerClient() (*client.Client, error) {
 		glog.Infof("Multiple api servers specified.  Picking first one")
 	}
 	clientConfig.Host = s.APIServerList[0]
+
+	if s.ChaosChance != 0.0 {
+		clientConfig.WrapTransport = func(rt http.RoundTripper) http.RoundTripper {
+			seed := chaosclient.NewSeed(1)
+			// TODO: introduce a standard chaos package with more tunables - this is just a proof of concept
+			// TODO: introduce random latency and stalls
+			return chaosclient.NewChaosRoundTripper(rt, chaosclient.LogChaos, seed.P(s.ChaosChance, chaosclient.ErrSimulatedConnectionResetByPeer))
+		}
+	}
+
```

Comment:
Small ask, could you put this into a separate method with a "chaos" description? Again, just to help organize the stuff that is for test purposes only.